HtmlCssSample.css

```diff
@@ -0,0 +1,199 @@
+.main-body {
+    background-color: rgb(35, 56, 76);
+    display: flex;
+    margin: 0;
+}
+
+.navigation-bar {
+    position: fixed;
+    justify-content: space-between;
+    background-color: white;
+    height: 7%;
+    width: 100%;
+    display: flex;
+}
+
+.logo {
+    display: flex;
+    justify-content: flex-start;
+    margin-left: 2%;
+    margin-top: 1%;
+    width: 10%;
+    height: 50%;
+}
+
+.webpage-main-logo {
+    height: 100%;
+}
+
+.navigation-element {
+    display: inline-flex;
+    justify-content: space-evenly;
+    width: 60%;
+    height: 100%;
+}
+
+.navigation-links {
+    display: flex;
+    justify-content: flex-end;
+    list-style-type: none;
+    margin-top: 2%;
+    width: 65%;
+    height: 30%;
+}
+.navigation-links-element
+{
+    display: inline-block;
+    padding-right: 2%;
+    padding-left: 2%;
+}
+.navigation-links-element:hover {
+    background-color: rgb(235, 233, 233);
+    cursor: pointer;
+}
+
+.navigation-bar-buttons {
+    margin-top: 2%;
```

Comment:
Instead of using the margin top, use the flex properties like Justify-content center and align-items center

Reply:
Resolved